swap/src/storage.rs

```diff
@@ -81,13 +81,16 @@ mod tests {
     use rand::rngs::OsRng;
     use serde::{Deserialize, Serialize};
     use std::str::FromStr;
-    use xmr_btc::serde::{bitcoin_amount, monero_private_key};
+    use xmr_btc::{
+        serde::{bitcoin_amount, monero_private_key},
+        CrossCurveScalar, Curve25519Scalar,
+    };
 
     #[derive(Debug, Serialize, Deserialize, PartialEq)]
     pub struct TestState {
         A: xmr_btc::bitcoin::PublicKey,
         a: xmr_btc::bitcoin::SecretKey,
-        s_a: xmr_btc::cross_curve_dleq::Scalar,
+        s_a: CrossCurveScalar,
```

Comment:
I have been very vocal about my dislike for using fully qualified paths but it is the approach we have settled upon, this patch goes against all the other repos. Whilst I'm not against this change it introduces non-uniformity across the codebase, both in this repo and in the rest of the CoBloX repos. In my opinion this change should be discussed on the mailing list and a uniform approach be applied.

Reply:
I can drop this commit for now. Upon second inspection the re-exports should be renamed anyway to cross_curve::Scalar and curve25519::Scalar to be consistent with bitcoin::Amount and monero::Amount etc